Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var array
+     */
+    private static $formats = array('json', 'xml', 'yml');
```

Comment:
> Yeah but can you be sure the constructor is called before the call to the static method happens ?

Upps, of course not. Actually that was the reason I set the array when defining the property. Then I was confused when you told somewhere else :)